projects/msc/README.md

```diff
@@ -0,0 +1,47 @@
+# Beyond Goldfish Memory: Long-Term Open-Domain Conversation
```

Comment:
@jxmsML let's actually pull this out of the PR and do a separate one with the project page (which we'll do at 9:25/9:30am tomorrow morning)